benchmark_runner/common/oc/oc.py

```diff
@@ -242,6 +242,61 @@ def get_prom_token(self):
         long_uuid = self.run(f"oc -n openshift-monitoring sa get-token prometheus-k8s")
         return long_uuid
 
+    @typechecked
+    def exec(self, command: str, pod_name: str, namespace: str=''):
+        """
+        oc exec a command and return the answer
+        :return:
+        """
+        try:
+            if namespace != '':
+                namespace = f'-n {namespace}'
+            return self.run(f'oc exec {namespace} {pod_name} -- {command}')
+        except Exception as err:
+            raise ExecFailed(command, pod_name, err)
+
+    @typechecked
+    def terminate_pod_sync(self, pod_name: str,
```

Comment:
We have already 'delete_pod_sync'

Reply:
I can't use that, because it requires a YAML file so that it can delete it.  Again, it's intended to clean up our pods, not to terminate an arbitrary system pod.